website/docs/support/known-issues.md

```diff
@@ -26,3 +26,5 @@ included in backup and restore.
 * SharePoint document library data can't be restored after the library has been deleted.
 
 * Sharing information of items in OneDrive/SharePoint using sharing links aren't backed up and restored.
+
+* Partial inheritance of link shares from a parent folder can't be restored.
```

Comment:
Imagine the following scenario where we have `folder` and `folder/file`:
- Add perm1 to `folder`
- Add perm2 to `file` but with inheritance disabled
- Add perm3 to `folder`

Actual links shares for `file`:
- perm2
- perm3

After backup and restore:
- perm2

For us to be able to do this, we have to be able to restore parent permissions after child IIUC as we don't have a way to "delete" inherited link shares as of now.

Reply:
"If a link share is created for an item with inheritance disabled (via the Graph API), Corso does not restore any link shares on that item that may be inherited after that"